poetry.lock

```diff
@@ -2950,6 +2963,7 @@ prometheus-client = [
     {file = "prometheus_client-0.14.1-py3-none-any.whl", hash = "sha256:522fded625282822a89e2773452f42df14b5a8e84a86433e3f8a189c1d54dc01"},
     {file = "prometheus_client-0.14.1.tar.gz", hash = "sha256:5459c427624961076277fdc6dc50540e2bacb98eebde99886e59ec55ed92093a"},
 ]
+plugin-with-view-overrides = []
```

Comment:
Does this lock file need updating? I see the packaged named both `plugin-with-view-overrides` and `plugin-with-view-override`.